homeassistant/components/recorder/table_managers/statistics_meta.py

```diff
@@ -157,9 +178,16 @@ def _update_metadata(
         This call is not thread-safe and must be called from the
         recorder thread.
         """
+        if "mean_type" not in new_metadata:
+            # To avoid breaking change as we added mean_type in schema version 49
+            # Even when typing suggests it is always present, we need to check it and guard
+            # against it as custom integrations might not set it.
```

Comment:
Can you try to rephrase so it's clearer how the extra check avoids a a breaking change?

Reply:
Better now?